frontend/src/v5/ui/routes/dashboard/projects/containers/uploadFileForm/uploadFileForm.component.tsx

```diff
@@ -179,61 +180,63 @@ export const UploadFileForm = ({ openState, onClickClose }: IUploadFileForm): JS
-									</DashboardListHeaderLabel>
-									<DashboardListHeaderLabel key="progress" width={297} hidden={!isUploading}>
-										<FormattedMessage id="uploads.list.header.progress" defaultMessage="Upload Progress" />
-									</DashboardListHeaderLabel>
-								</UploadsListHeader>
-							</div>
 							{!!fields.length && (
-								<UploadList
-									values={sortedList}
-									selectedIndex={selectedIndex}
-									isUploading={isUploading}
-									onClickEdit={(id) => onClickEdit(id)}
-									onClickDelete={(id) => onClickDelete(id)}
-									getOriginalIndex={getOriginalIndex}
-								/>
+								<>
+									<UploadsListHeader
+										onSortingChange={setSortConfig}
+										defaultSortConfig={DEFAULT_SORT_CONFIG}
+									>
+										<DashboardListHeaderLabel key="file" name="file.name" minWidth={122}>
+											<FormattedMessage id="uploads.list.header.filename" defaultMessage="Filename" />
+										</DashboardListHeaderLabel>
+										<DashboardListHeaderLabel key="destination" width={352}>
+											<FormattedMessage id="uploads.list.header.destination" defaultMessage="Destination" />
+										</DashboardListHeaderLabel>
+										<DashboardListHeaderLabel key="revisionName" width={isUploading ? 359 : 399}>
+											<FormattedMessage id="uploads.list.header.revisionName" defaultMessage="Revision Name" />
+										</DashboardListHeaderLabel>
+										<DashboardListHeaderLabel key="progress" width={297} hidden={!isUploading}>
+											<FormattedMessage id="uploads.list.header.progress" defaultMessage="Upload Progress" />
+										</DashboardListHeaderLabel>
+									</UploadsListHeader>
+									<UploadList
+										values={sortedList}
+										selectedIndex={selectedIndex}
+										isUploading={isUploading}
+										onClickEdit={(id) => onClickEdit(id)}
+										onClickDelete={(id) => onClickDelete(id)}
+										getOriginalIndex={getOriginalIndex}
+									/>
+								</>
 							)}
 							<DropZone
 								message={formatMessage(
 									{ id: 'uploads.dropzone.message', defaultMessage: 'Supported file formats: IFC, RVT, DGN, FBX, OBJ and <MoreLink>more</MoreLink>' },
-									{ MoreLink: (child: string) => <a href="https://help.3drepo.io/en/articles/4798885-supported-file-formats" target="_blank" rel="noreferrer">{child}</a> },
+									{ MoreLink:
+										(child: string) => (
+											<a href="https://help.3drepo.io/en/articles/4798885-supported-file-formats" target="_blank" rel="noreferrer">{child}</a>
+										),
+									},
 								)}
 								processFiles={(files) => { processFiles(files); }}
 								hidden={isUploading}
 							/>
 						</Padding>
-					</ScrollArea>
+					</UploadsListScroll>
 					<Sidebar
-						open={Number.isInteger(selectedIndex) && !isUploading}
+						open={sidebarOpen}
 						onClick={() => setSelectedIndex(null)}
-						noButton={!(Number.isInteger(selectedIndex))}
+						noButton={!sidebarOpen}
```

Comment:
After the changes, `noButton` is true when the `sidebarOpen` is false.
I think you can get rid of it then